kernelci/cli/user.py

```diff
@@ -115,127 +110,94 @@ def add(username, email, config,  # pylint: disable=too-many-arguments
         'username': username,
         'email': email,
         'password': password,
-        'groups': group
     }
     configs = kernelci.config.load(config)
     api_config = configs['api'][api]
     api = kernelci.api.get_api(api_config, secrets.api.token)
-    data = api.create_user(user)
-    click.echo(json.dumps(data, indent=indent))
```

Comment:
I think we should keep this to let the user know that a new user has been added.

Reply:
The command provides the user name so it's not necessary to print anything as a result.  We might want to add a verbose flag, but I think it just complicates the code for nothing.  If you want to check what was done after `kci user add bob` you can do `kci user find username=bob`.

Reply:
Okay.